src/check_defines_custom.h

```diff
@@ -0,0 +1,70 @@
+#ifndef CHECK_DEFINES_CUSTOM_H
```

Comment:
Better to have this file in the CustomBuilds folder

Reply:
Done